test/exchange-files.js

```diff
@@ -61,7 +61,7 @@ const dirs = [
 
 const timeout = 240 * 1000
 
-describe.skip('exchange files', () => {
```

Comment:
Why were these disabled????! 👀

Reply:
check this PR [interop#42](https://github.com/ipfs/interop/pull/42)

Reply:
Agree! Created an issue for tracking it [ipfs/interop#45](https://github.com/ipfs/interop/issues/45).